rootconf/default/lib/onie/functions

```diff
@@ -44,33 +46,33 @@ syslog_tag="onie"
 
 _log_msg()
 {
-	printf "$@" | tee $tee_log_file | logger -t $syslog_tag -p ${syslog_onie}.info
+	printf "$@" | tee -a $tee_log_file | logger -t $syslog_tag -p ${syslog_onie}.info
 }
 
 _log_console_msg()
 {
-	printf "$@" | tee /dev/console | logger -t $syslog_tag -p ${syslog_onie}.info
+	printf "$@" | tee -a $console_log_file | logger -t $syslog_tag -p ${syslog_onie}.info
```

Comment:
It seems we could just use `$tee_log_file` here.  Then we do not need `$console_log_file`.

Thinking a bit about names.

The name `tee_log_file`, which I wrote initially, is not a very good name.  It does not tell the programmer what the file is really for.  If we do consolidate on one log file, I think `$console_log_file` is a better name.   I would search and replace `$tee_log_file` with `$console_log_file`.  That name more clearly indicates what the file is for.